lib/rouge/formatters/html.rb

```diff
@@ -26,7 +26,7 @@ def safe_span(tok, safe_val)
         end
       end
 
-    private
+
```

Comment:
Oh, I didn't know it. There are too many `private*` scopes in Ruby ;)

However, ruby 1.9.3 does not support `private_constant`: https://ruby-doc.org/core-1.9.3/Module.html

It seems that 2.0.0 support it: https://ruby-doc.org/core-2.0.0/Module.html

IMO we can drop 1.9.3 support, while 2.0.0 is the default version of macOS Sierra.